addons/web/static/src/scss/bootstrap_review.scss

```diff
@@ -39,6 +39,14 @@
         // TODO remove me in master.
         background-color: transparent !important;
     }
+    table {
+        color: inherit;
+    }
+}
+// Card BG is white by default, color should be the opposite
+$card-color: color-yiq($card-bg) !default;
+.card {
+    color: $card-color;
```

Comment:
The comment does not help me understand why this is needed: what you say is already the case without this rule (thanks to the o-bg-color call on the card-body which already does that) ?